spark-connector/spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/SparkTransformConverter.java

```diff
@@ -224,6 +290,38 @@ public static IdentityTransform createSparkIdentityTransform(String columnName)
     return IdentityTransform.apply(Expressions.column(columnName));
   }
 
+  public static HoursTransform createSparkHoursTransform(String[] fieldName) {
+    return LogicalExpressions.hours(
+        Expressions.column(String.join(ConnectorConstants.DOT, fieldName)));
+  }
+
+  public static BucketTransform createSparkBucketTransform(int numBuckets, String[] fieldNames) {
+    return LogicalExpressions.bucket(numBuckets, createSparkNamedReference(fieldNames));
+  }
+
+  public static DaysTransform createSparkDaysTransform(String[] fieldName) {
+    return LogicalExpressions.days(
+        Expressions.column(String.join(ConnectorConstants.DOT, fieldName)));
```

Comment:
use `getFieldNameFromGravitinoNamedReference` to replace `String.join(ConnectorConstants.DOT, fieldName)`?

Reply:
updated.